main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md

```diff
@@ -1,163 +1,120 @@
 
-## Interaction with Stride Chain
+- `zcf`: Zoe Contract Facet.
+- `privateArgs`: Object containing remote references to various services.
+- `zone`: A `Zone` object with access to storage for persistent data.
+- `OrchestrationTools`: A set of orchestration related tools needed by the contract.
 
 ```js
-const stride = await orch.getChain('stride');
-const strideAccount = await stride.makeAccount();
+const contract = async (
+  zcf,
+  privateArgs,
+  zone,
+  { orchestrateAll, zcfTools }
+) => {
+  const { unbondAndTransfer } = orchestrateAll(flows, { zcfTools });
+
+  const publicFacet = zone.exo('publicFacet', undefined, {
+    makeUnbondAndTransferInvitation() {
+      return zcf.makeInvitation(
+        unbondAndTransfer,
+        'Unbond and transfer',
+        undefined,
+        harden({
+          give: {},
+          want: {},
+          exit: M.any()
+        })
+      );
+    }
+  });
+
+  return harden({ publicFacet });
+};
 ```
 
-### Get Chain
+The `orchestrateAll` function links the flows from the flows file to the contract logic. In this case, it links the `unbondAndTransfer` flow. The `publicFacet` exposes the `makeUnbondAndTransferInvitation` method, which creates a Zoe invitation to allow users to make an offer for the unbonding and transferring process.
 
-Retrieves the stride chain object using the orchestrator.
-
-### Make Account
-
-Creates an account on the stride chain.
-
-## `contract` Function
-
-The `contract` function when wrapped inside `withOrchestration` defines the [`start` function](#start-function) which is the entry point of the contract. The contract exports a `start` function [below](#start-function). It is merely a convention/convenience that we define a more abstract `contract` function here and pass it to `withOrchestration`. The arguments of this function are `zcf`, `privateAge`, `zone`, and `tools` for orchestration.
+The following code defines the `start` function of the contract that is returned by a call to `withOrchestration` with [`contract` function](#contract-function) as a parameter. In essence `contract` function is the entry point or `start` function of this contract with some orchestration setup.
 
 ```js
-/**
- * Orchestration contract to be wrapped by withOrchestration for Zoe
- *
- * @param {ZCF} zcf
- * @param {{
- *   agoricNames: Remote<NameHub>;
- *   localchain: Remote<LocalChain>;
- *   orchestrationService: Remote<CosmosInterchainService>;
- *   storageNode: Remote<StorageNode>;
- *   marshaller: Marshaller;
- *   timerService: Remote<TimerService>;
- * }} privateArgs
- * @param {Zone} zone
- * @param {OrchestrationTools} tools
- */
-const contract = async (zcf, privateArgs, zone, { orchestrate }) => {
+export const start = withOrchestration(contract);
 ```
 
-### `contract` Function Parameters:
+---
```

Comment:
Are the double section separators intentional? They look weird.
<img width="816" alt="Screenshot 2024-10-02 at 4 01 16 AM" src="https://github.com/user-attachments/assets/7874713c-aba6-4b4c-8be7-8c0ae5aa66d6">

Reply:
No - let me fix that.